CHANGELOG.txt

```diff
@@ -13,6 +13,7 @@
 * Added frontend integration tests (written with Casper JS)
 * Removed frontend integration tests (written with Selenium)
 * Provides the ability to declare cache expiration periods on a per-plugin basis
+* Removes disabled action button border
```

Comment:
i think we don't need to be too specific. we've done some other ui/ux improvements as wel, we can probably just put "Minor UI improvements" here /cc @FinalAngel